CHANGELOG.md

```diff
@@ -1,5 +1,58 @@
 # Release Notes
 
+## Version 1.12.2 - 2020-08-03
+
+Features:
+* Automated GCP Credentials Addon Prototype [#8682](https://github.com/kubernetes/minikube/pull/8682)
+* status: Add experimental cluster JSON status with state transition support [#8868](https://github.com/kubernetes/minikube/pull/8868)
+* Add support for Error type to JSON output [#8796](https://github.com/kubernetes/minikube/pull/8796)
+* Implement Warning type for JSON output [#8793](https://github.com/kubernetes/minikube/pull/8793)
+* Add stopping as a possible state in deleting, change errorf to warningf [#8896](https://github.com/kubernetes/minikube/pull/8896)
+* Use preloaded tarball for cri-o container runtime [#8588](https://github.com/kubernetes/minikube/pull/8588)
+
+Bug Fixes:
+* docker/podman: warn if allocated memory is below limit [#8718](https://github.com/kubernetes/minikube/pull/8718)
+* Make sure to prepare the kic volume after creation [#8777](https://github.com/kubernetes/minikube/pull/8777)
+* Generate cri-o container runtime preload tarball [#8581](https://github.com/kubernetes/minikube/pull/8581)
+* Enabling metrics addon when someone enables dashboard [#8842](https://github.com/kubernetes/minikube/pull/8842)
+* make base-image respect --image-repository [#8880](https://github.com/kubernetes/minikube/pull/8880)
+* restart kubelet if config changes [#8817](https://github.com/kubernetes/minikube/pull/8817)
```

Comment:
re-word this to a way user would recongize it... like what was the Error we fixed.
"Fix upgrading from old minkube versions"

Reply:
done